composer.json

```diff
@@ -48,6 +48,7 @@
         "react/http": "^1.5",
         "symfony/console": "^5",
         "symfony/polyfill-php80": "^1.22",
+        "symfony/polyfill-php81": "^1.26",
```

Comment:
Do we need that requirement here in this PR?

I think it is required anyways, for example:

symfony/dependency-injection  v5.4.20  requires  symfony/polyfill-php81 (^1.22)
and
web-token/jwt-framework   v2.2.11  requires   symfony/dependency-injection (^4.2|^5.0)
and
magento/product-community-edition     2.4.5-p1  requires  web-token/jwt-framework (^v2.2.7)


So when you have a magento installation you will have polyfill-php81 (since magento 2.4.4), or I am wrong somewhere?